scripts/gulp/tasks/bundle.ts

```diff
@@ -22,6 +22,10 @@ task('bundle:package:clean', () => del([`${paths.packageDist(packageName)}`], {
 
 const componentsSrc = [paths.packageSrc(packageName, '**/*.{ts,tsx}')];
 
+task('copy:readme', function () {
+  return src(paths.packages('', 'README.md')).pipe(dest(paths.packages('react-northstar')));
```

Comment:
I think you'll also need to allow the file to be published in the `files` entry of the `package.json`

Reply:
Added